### ISSUE TITLE
Saving StoryProgressView state 

### ISSUE THREAD

**Issue by gulzar1996** [shts/StoriesProgressView#6]
How can I save the ProgressView state when used in a viewpager so that I can restore the state when switching back?

**Comment by darwinPro** [comment]
I agree, it would be great if you could start from an unfinished number of storie

**Comment by shts** [comment]
@gulzar1996 @darwinPro 
update to new version 3.0.0. You can use `startStories(int)`.
